launch/airsim_main.launch

```diff
@@ -3,8 +3,12 @@
     <arg name="enable_autonomy" default="true"/>
     <arg name="enable_exploration" default="true"/>
 
+    <!-- Which nodes -->
+    <arg name="do_slam" default="true"/>
```

Comment:
I would think that most people wouldn't need SLAM by default, and would want to keep the default 'false'

Reply:
Perhaps, but the main advantage of our AirSim wrapper over others is that SLAM is already set up (that and it works well with MAVROS), so I'd rather keep it on by default